docs/README.md

```diff
@@ -59,11 +59,40 @@ Both `conseil` and `lorre` write to `syslog`. The logs are verbose enough to det
 
 ### Configuration
 
-`conseil` and `lorre` use [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format to store configuration. Read more at [Typesafe Config] github page(https://github.com/lightbend/config)
+`conseil` and `lorre` use [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format to store configuration. Read more at [Typesafe Config](https://github.com/lightbend/config) github page.
 
-### Datasource configuration
+### Metadata overrides
+
+Metadata enables dynamic querying of data sources provided by Conseil and allows for chain-agnostic UIs, like Arronax, to be built while still providing a high level of specialization. Some of the metadata override keys are helpful for query construction, but a large number of them exist for rendering.
+
+In addition to the metadata derived from the schema, it's possible to extend and override it. This information is stored in configuration files inside `/src/main/resources/metadata/` which are referenced from `/src/main/resources/metadata.conf` that is itself included from `src/main/resources/reference.conf`. Metadata hierarchy is platform/network/entity/attribute, different options are available at each level.
+
+Note the naming convention in this file. Hyphenated keys become camel-cased when presented via the metadata service in JSON format. For example `display-name` in the config file appears as `displayName` in the JSON response.
+
+#### Common metadata
+
+These overrides are available at all levels.
+
+- `description`: Item description.
+- `display-name`: Item name.
+- `visible`: Item visibility, default is `false`. This means when a new entity is added to the Conseil database, it must be explicitly enabled.
+
+#### Entity metadata
+
+- `display-name-plural`: Arronax uses this to render entity tab titles here it will show "Blocks" instead of "Block".
+
+#### Attribute metadata
+
+- `currency-symbol`: Associated with Currency type fields, contains currency symbol or code, for example, 'XTZ', or 'ꜩ'
+- `currency-symbol-code`: Same as `currency-symbol`, but stores the Unicode index of the currency symbol, for 'ꜩ' it would be `42793`
+- `data-format`: Presently used to customize date format. String contents of this item have no impact on how the data is sent or queried, it would be up to the implementing UI to interpret this content.
```

Comment:
Should this entry actually show what kind of standard notation is used to define the formats?
Possibly improve for another doc iteration

Reply:
another iteration, the conseil docs need a deeper update